server/controllers/jobs_controller.go

```diff
@@ -48,17 +51,39 @@ func (j *JobsController) GetProjectJobs(w http.ResponseWriter, r *http.Request)
 		CleanedBasePath: j.AtlantisURL.Path,
 	}
 
-	if err = j.ProjectJobsTemplate.Execute(w, viewData); err != nil {
+	return j.ProjectJobsTemplate.Execute(w, viewData)
+}
+
+func (j *JobsController) GetProjectJobs(w http.ResponseWriter, r *http.Request) {
```

Comment:
Should this also be lowercase?
```suggestion
func (j *JobsController) getProjectJobs(w http.ResponseWriter, r *http.Request) {
```

Reply:
no, these should be exported ideally to represent that only this function should be called since it has stats associated with it.